frontend/src/components/resources/stack/info.tsx

```diff
@@ -205,50 +205,77 @@ export const StackInfo = ({
         latest_contents.length > 0 &&
         latest_contents.map((content) => {
           const showContents = show[content.path] ?? default_show_contents;
+          const handleToggleShow = () => {
+            setShow({ ...show, [content.path]: !showContents });
```

Comment:
```suggestion
            setShow((show) => ({ ...show, [content.path]: show[content.path] }));
```
Avoids sync edge cases of having this function capture showContents from outside